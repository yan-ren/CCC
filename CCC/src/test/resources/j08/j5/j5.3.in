5
7 6 7 6
7 3 1 3
7 7 4 8
7 6 7 5
6 5 7 6
