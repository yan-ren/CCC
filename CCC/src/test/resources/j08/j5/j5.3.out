Roland
Roland
Patrick
Patrick
Roland
